src/analytics/Properties.tsx

```diff
@@ -1614,7 +1614,11 @@ interface EarnEventsProperties {
   } & EarnCommonProperties
   [EarnEvents.earn_deposit_add_gas_press]: EarnCommonProperties & { gasTokenId: string }
   [EarnEvents.earn_feed_item_select]: {
-    origin: 'EarnDeposit' | 'EarnWithdraw' | 'EarnClaimReward' | 'EarnSwapDeposit'
+    origin:
+      | TokenTransactionTypeV2.EarnDeposit
+      | TokenTransactionTypeV2.EarnWithdraw
+      | TokenTransactionTypeV2.EarnClaimReward
+      | TokenTransactionTypeV2.EarnSwapDeposit
```

Comment:
@satish-ravi do you think this could be a problem?
The value will be different now.

Reply:
I guess we generally try to avoid breaking changes for analytics in case there's some dashboard dependencies. We have had instances in the past where we had to workaround to avoid this. But maybe fine here, I don't think this is a critical analytics event. Would be good to confirm with product

Reply:
Thanks yes I'll check.